scripts/validate_schemas.js

```diff
@@ -66,16 +65,23 @@ module.exports = {
 // @ts-ignore
 if (require.main === module) {
   // When this is being called as a script via `node scripts/validate_schemas.js`
-  getBreakingChanges().then(changes => {
-    if (changes.length) {
-      process.exitCode = 1
-      console.error(
-        "Failing due to breaking changes between Force and Metaphysics Production\n\n"
-      )
-      console.error(changes)
-      console.error(
-        "\n\nYou should deploy metaphysics production, and re-deploy force"
-      )
-    }
-  })
+  if (process.argv.length != 3) {
+    console.log('This script must be called with either "staging" or "production"')
+    process.exitCode = 1
+  } else {
+    getBreakingChanges(process.argv[2]).then(changes => {
```

Comment:
totally minor, but I'd recommend extracting a local variable for `process.argv[2]` considering that it's referenced twice in this script

Reply:
👍